backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -133,7 +133,7 @@
       <div class="col-12 col-lg-4">
         <div class="form-group">
           <%= label_tag :q_store_id_in, Spree.t(:store) %>
-          <%= f.select :store_id_in, Spree::Store.order(:name).pluck(:name, :id), { include_blank: true }, class: 'select2-clear js-filterable' %>
+          <%= f.select :store_id_in, @stores.pluck(:name, :id), { include_blank: true }, class: 'select2-clear js-filterable' %>
```

Comment:
`<%= f.collection_select(:store_id_in, @stores, :id, :unique_name,  { include_blank: true }, { class: 'select2-clear js-filterable' }) %>`

Would `:unique_name` be better?

Reply:
@MatthewKennedy good point, although we'll remove this dropdown soon, @Vegann is working on it :)

Reply:
Ahhh yes, the admin by store, looking forward to that addition to spree!